src/main/java/org/apache/mesos/hdfs/executor/AbstractNodeExecutor.java

```diff
@@ -173,10 +170,10 @@ protected void startProcess(ExecutorDriver driver, Task task) {
    * Reloads the cluster configuration so the executor has the correct configuration info.
    **/
   protected void reloadConfig() {
-    if (!reloadLimiter.tryAcquire()) {
-      log.info("Limiting reload rate");
-      return;
-    }
+//    if (!reloadLimiter.tryAcquire()) {
```

Comment:
Yes, I prefer removing commented out code. Putting a TODO in place sounds good if needed, but I think you already have one in the reloadConfig method. Thx!